commands/inventory/add_service_mysql.go

```diff
@@ -29,8 +27,12 @@ MySQL Service added.
 Service ID     : {{ .Service.ServiceID }}
 Service name   : {{ .Service.ServiceName }}
 Node ID        : {{ .Service.NodeID }}
+{{ if .Service.Socket }}\
+Socket         : {{ .Service.Socket }}
+{{else}}\
```

Comment:
I think `\` is a kingpin's extension we don't use anywhere. Go templates added support for `{{-` / `-}}` for the same (or similar) purpose: https://golang.org/pkg/text/template/#hdr-Text_and_spaces. Can we use this Go feature?